api/src/test/java/org/openmrs/module/dhisreport/api/adx/AdxTypeTest.java

```diff
@@ -0,0 +1,107 @@
+package org.openmrs.module.dhisreport.api.adx;
+
+import static org.junit.Assert.*;
```

Comment:
We also do not encourage using wild card imports as per https://wiki.openmrs.org/display/docs/Coding+Conventions